### DISCUSSION TITLE
Splash Screen tutorial

### DISCUSSION THREAD

**Discussion by AUSoftware** [unoplatform/uno#14621] [Q&A]
The tutorial of Splash Screen (https://platform.uno/docs/articles/splash-screen.html?tabs=tabid-vswin) seems not working for the new Uno template. The sample (https://github.com/unoplatform/Uno.Samples/tree/master/UI/SplashScreenSample) works but it looks like on old template.
I have error when trying building Android: _Error	APT2260	resource drawable/assets_splashscreen (aka com.companyname.UnoApp1:drawable/assets_splashscreen) not found_. I published my works at  https://github.com/AUSoftware/UnoApp1

Could you please update the tutorial for the new template?

Thanks

**Comment by jeromelaban** [comment]
@pictos may be able to help

**Comment by pictos** [comment]
I'm out of energy, so as soon it gets back I'll take a look on it and provide guidance and see how I can update the docs

**Comment by pictos** [comment]
@AUSoftware I believe there's a misunderstanding on how you can set up your Splashscreen. So, if you want to use `Uno.Resizetizer` to setup the Splashscreen for you, all you need to do is provide your asset as `svg` and change the value of `UnoSplashScreen` on your `base.props` to match your asset name and location, you can find more details on this [doc](https://platform.uno/docs/articles/external/uno.resizetizer/doc/using-uno-resizetizer.html?tabs=classlib%2CAndroid#unosplashscreen)

Now, if you want to handle the Splashscreen by yourself, maybe your scenario isn't supported by `Uno.Resizetizer` or you already have all the assets needed, you must follow the docs that you mentioned and make sure: 
- To not use `UnoSplashScreen`
- Provide all the image sizes and place them in the correct folder

Does it makes sense to you? I'll make sure to update that doc and mention `Uno.Resizetizer` and how to remove it.
